stytra/experiments/tracking_experiments.py

```diff
@@ -31,6 +31,7 @@
 from stytra.hardware.video.write import H5VideoWriter, StreamingVideoWriter
 
 import sys
+import os
```

Comment:
minor observation, but for consistency I would use the `pathlib `module instead of `os`

Reply:
Resolved in a5681ac181c3c4aae93217277cf32293b3ce0cd3